server/templates/place.html

```diff
@@ -17,7 +17,15 @@
 
 {% set main_id = 'dc-places' %}
 {% set page_id = 'page-dc-places' %}
-{% set title = place_name + ' ' + category + ' - ' + _('Place Explorer') %}
+{% set category_title = place_name + ' ' + category %}
+{% set overview_title = place_name + ' - Economics, Health, Equity, Crime, Education, Demographics, Housing, Environment' %}
+{% if category != '' %}
+  {% set title = category_title %}
+  {% set description = 'Statistics about ' + place_name + ' ' + category %}
```

Comment:
Add localization

Reply:
Added localization plumbing to the page. Translations won't show up until after the PR goes in and we extract and submit these strings to Google's translation pipeline.